src/components/FestivalCardLarge/FestivalCardLarge.tsx

```diff
@@ -1,38 +1,67 @@
-import Button from "../Buttons/Button";
 import styles from "./FestivalCardLarge.module.css";
 
-function FestivalCardLarge() {
+export type FestivalCardLargeProps = {
+  name: string;
+  location: string;
+  begin: string;
+  end: string;
+  visitors: number;
+  acts: number;
+  price: number;
+  allacts: string;
+  website: string;
```

Comment:
you could wrap these props into a content object, then you can simply hand over the Festival object in a single prop